### PR TITLE
PLT-890 Update debian package naming convention

### PR DESCRIPTION
**Pull request by elopatin-plos**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
   <artifactId>named-entity-service</artifactId>
   <packaging>war</packaging>
   <name>Named Entity JAX-RS Service</name>
-  <version>1.6.0-SNAPSHOT</version>
+  <version>1.6.0</version>
```
Comment (mbowen) [inline pom.xml]: this should be -SNAPSHOT.

File: pom.xml [hunk 2]
```diff
@@ -496,7 +499,7 @@
               <goal>jdeb</goal>
             </goals>
             <configuration>
-              <deb>${project.build.directory}/${artifactId}-[[version]].deb</deb>
+              <deb>${project.build.directory}/${project.artifactId}_[[version]]+${build.timestamp}-plos${build.counter}.deb</deb>
               <!--<verbose>true</verbose>-->
               <!-- replace SNAPSHOT in version with current timestamp -->
               <snapshotExpand>true</snapshotExpand>
```
Comment (mbowen) [inline pom.xml]: maybe should comment this out.

File: src/deb/control/control [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Package: [[artifactId]]
 Version: [[version]]
-Description: [[name]]
+Description: [[name]]+[[build.timestamp]]-plos[[build.counter]]
 Architecture: all
```
Comment (jonocodes) [inline src/deb/control/control]: Should this be in the version field or the description field?

Comment (jonocodes) [inline src/deb/control/control]: It looks to me like dipro is using the version field: https://github.com/PLOS/content-repo/pull/80/files

Comment (elopatin-plos) [inline src/deb/control/control]: Oops – my bad. Thanks for catching that; will change it to use the version field shortly (for both NED and NED-CAS).

Comment (elopatin-plos) [inline src/deb/control/control]: done

**Pull request merged**